lib/ssf/ssf_methods.rb

```diff
@@ -23,6 +23,7 @@ def finish(time: nil)
     end
 
     def child_span(operation: '', tags: {})
+      puts "WARN: this method is deprecated."
```

Comment:
Any reason for not just removing this entirely? (I'm not sure if we really even have any other users at this stage besides us :) )